receiver/hostmetricsreceiver/example_config.yaml

```diff
@@ -4,8 +4,10 @@ extensions:
 
 receivers:
   hostmetrics:
-    default_collection_interval: 10s
+    default_collection_interval: 1s
```

Comment:
this should be 60s, just because users will copy-paste this, so it is better to be more realistic